README.md

```diff
@@ -32,4 +32,4 @@ Saeed | Sarkarati | **Photonics/Physics, PhD** | Physics, Programming, Teaching,
 Jafar | Khaksar | **Economics, MSc student** | Quantitative Economics, Econometrics, Data Science, Julia 
 Mobin | Moradi | **Physics, BSc** | Physics, Mathematics
 Mehdi | Mirzaie | **Mathematics. Ph.D.** | Data Science, Bioinformatics
-Mohammad | Ebadi | None | None
+Mohammad | Ebadi | **Associate degree**	 | Programming , Web designing , Cybersecurity , Network 
```

Comment:
@Mohammad-develope 
Change Web designing to Web Desgin
Change Networ to Networking

Reply:
@Mohammad-develope 
Also change the "degree" to "Degree"

Reply:
Sure.